GoogleForCreators/web-stories-wp
# Design Panel: Improve accessibility for panel headers

## Summary

For sections 'Size & position', 'Layer', 'Style', 'Link'
1.disable navigation to each header's A11Y subtree 
2.Add aria-expanded to each section header

## Relevant Technical Choices

Move the aria-* labels from the nested button to the header

## To-do

N/A

## User-facing changes

N/A

## Testing Instructions

1. Enable VoiceOver
2. Open Web Stories and choose a Shape Element
3. Use the VO key to navigate the header
4. Make sure the voice will announce the status of the section header and skip over the nested button.
5. Sample announcement text could be `Size & position panel, expanded, button`

---

Fixes #4603
